test/test-framework/.test-osdk-scorecard.yaml

```diff
@@ -16,15 +16,3 @@ scorecard:
         csv-path: "deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml"
         proxy-image: "quay.io/example/scorecard-proxy"
         proxy-pull-policy: "Never"
-    - external:
-        command: scorecard/bin/print-json.sh
```

Comment:
These files under `test/test-framework/scorecard` can be removed to right?

Reply:
good find, I have removed them.